.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,107 @@
+name: CI Pipeline GitHub
+
+on: [push, pull_request]
+
+jobs:
+  backend-test:
+    runs-on: ubuntu-latest
+    env:
+      WORKDIR: backend
+      JWT_SECRET: ${{ github.sha }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+      - name: Install dependencies
+        run: cd $WORKDIR && npm install
+      - name: Run tests
+        run: cd $WORKDIR && npm run test:ci
+      - name: Archive test results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: junit-report-backend
+          path: ${{ env.WORKDIR }}/junit.xml
+
+  frontend-test:
+    runs-on: ubuntu-latest
+    env:
+      WORKDIR: frontend
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+      - name: Install dependencies
+        run: cd $WORKDIR && npm install
+      - name: Run tests
+        run: cd $WORKDIR && npm run test
+      - name: Archive test results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: junit-report-frontend
+          path: ${{ env.WORKDIR }}/dist/junit-report.xml
+
+  backend-package:
+    runs-on: ubuntu-latest
+    services:
+      docker:
+        image: docker:25.0.3-dind
+        options: --privileged
```

Comment:
By using this [github documentions](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images) it seems like we could remove this service